meetings/2024-08-21.md

```diff
@@ -44,11 +49,51 @@
 ### Solid Community AU follow-up
 
 * VB: STM to follow up on specific topics on August 27th or September 3rd (12:00 UTC).
-
+* VB: Related to encryption and authorization.
+* eP: Discovery.
+* SC: Message payload being encrypted? Storage encryption?
+* eP: I understood it as resource being encrypted.
 
 ### Rui Zhao's work
 
 * eP: invited Rui to present his work
-paper: https://dl.acm.org/doi/10.1145/3589334.3645631
-draft: https://renyuneyun.github.io/solid-dtou/dtou-spec.html
-mentioned in https://github.com/solid/security-considerations/issues/12#issuecomment-2191897863
+  * paper: https://dl.acm.org/doi/10.1145/3589334.3645631
+  * draft: https://renyuneyun.github.io/solid-dtou/dtou-spec.html
+  * mentioned in https://github.com/solid/security-considerations/issues/12#issuecomment-2191897863
+* RZ: slides: https://docs.google.com/presentation/d/1Ax7Vjo0FkmguhhnHRbhj_iQ5ivQzZjs_urqHhixqu1o/edit
+* RZ: We don't like the "biggest lie on the Internet" but their contents are important.
+* RZ: Are we at a better position in Solid? Actually no. Problems are more serious. We expect more apps; user to make decision on data usage (autonomy). DToU may be a way out.
+* RZ: DTOU research targets: encoding user preferences (or data usage) and/or apps promises/behaviours (of data usage); supporting interoperability in decentralized context; check compliance and/or other policy semantics (e.g., obligations); support continous policy checking.
+* RZ: Perennial.
+* RZ: Users encode data  policy and associate with documents, and app policy to be sent by the app to Solid service, protocol to perform reasoning tasks and facilitate user.
+* RZ: Some advantages (compare to normal access control) to for users and apps. 
+* RZ: There are policy sets. We want to how input data/policy should generate output.
+* RZ: Needs policy authoring - user needs to be able to write data policies with helper tools, users need to write access rules, apps need to write app policies. Needs to go beyond document/container-level policy.
+* RZ: Need some helpers: ML/NLP/LLMe, e.g., privacy policy to app policy, user behaviour observation agent. Better semantic coverage (unifying tags, extended activation condition, time constraints). Align with, e.g., ODRL. Establish "contract", provenance and auditing tools.
+* eP: I've read your paper. You also have the spec but haven't read yet. Seems comprehensive.
+* eP: Are you familiar with some existing efforts - noting SC's mentions
+* eP: https://tosdr.org/
+* RZ: Right found that in the early days of the work. I wasn't focusing on that because that was still relying on natural language policies (c. 2018/2019). The only way to use formal encodings and that's where my research is heading.
```

Comment:
```suggestion
* RZ: Right, found that in the early days of the work. I wasn't focusing on that, because that was still relying on natural language policies (c. 2018/2019). The only way is to use formal encodings, and that's where my research is heading.
```